app/misc/git.ts

```diff
@@ -282,7 +282,9 @@ function getAllCommits(callback) {
         function (cb) {
           if (!refs[count].isRemote()) {
             console.log("referenced branch exists on remote repository");
-            repos.getReferenceCommit(refs[count])
+            refs[count].peel(Git.Object.TYPE.COMMIT) 
```

Comment:
Annotated tabs used to cause failures here because they are not simply a reference, like non-annotated tags are. The peel(Git.Object.TYPE.COMMIT) function isolates the commit reference, exactly what we want.

Reply:
Tthis is a good fix, I can open a testing repo with annotated tags now.